src/Core/Services/Implementations/LaunchDarklyFeatureService.cs

```diff
@@ -16,6 +16,7 @@ public class LaunchDarklyFeatureService : IFeatureService
     private readonly ICurrentContext _currentContext;
     private const string _anonymousUser = "25a15cac-58cf-4ac0-ad0f-b17c4bd92294";
 
+    private const string _contextKindDevice = "device";
     private const string _contextKindOrganization = "organization";
     private const string _contextKindServiceAccount = "service-account";
```

Comment:
At what point would we want to make this an enum/class?

Reply:
Well, I think I am done adding them and we can't invent any more, then another comes up. This is (probably) the last one! 😜